jsapp/js/account/plans/addOnList.component.tsx

```diff
@@ -59,6 +63,18 @@ const AddOnList = (props: {
     setAddOnProducts(addonProducts);
   }, [props.products]);
 
+  const currentPlan = useMemo(() => {
```

Comment:
This should get renamed to `currentAddOn`, since it doesn't use `subscriptionStore.planResponse`. 👍